### PR TITLE
New block images grid

### PR DESCRIPTION
**Pull request by kesiah**
Test URLs:
- Before: https://main--vg-volvotrucks-us-rd--netcentric.hlx.page/drafts/laura/
- After: https://11-our-promises--vg-volvotrucks-us-rd--netcentric.hlx.page/drafts/laura/

### CODE REVIEW

**Comment by cogniSyb** [comment]
Something has gone wrong with the test URLs. But I checked out the code and could run it locally :)

Regarding alignment, the design contains heading + text that’s always over two lines. Is it possible to add a soft enter in the document so that we have at least the headings spanning two lines?

In your example I miss the 5th image, the full width landscape image. Could you add this one?

Some linting errors should be fixed as well, mostly the class selectors should be written in kebab-case.

**Comment by kesiah** [comment]
> 
- URLS fixed!
- Added last fullwidth image, sorry I thought it was another component.
- Fixed linting issues
- Title in 2 lines: I would avoid adding `<br>` from the editor, cause in smaller viewport, we can get strange behaviour. So I can add more paddings, so the text width is decreased in bigger viewports, but currently I'm using the paddings that the figma design uses.

**Comment by kesiah** [comment]
> Maybe it’s good to leave the content as is indeed. Even though it’s not supported in every browser, we can add `text-wrap: balance;` to both the title and the text.
> 
> I see in viewport between 460px and 744px that the images aren’t filling the cards. Could the image get a `width: 100%` in here?

done!

File: blocks/images-grid/images-grid.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    cell.remove();
+  });
+
+  block.firstElementChild.append(ul);
+
+  // give format to the list items
+  [...ul.children].forEach((li) => {
+    const section = document.createElement('div');
+
+    // add link to the image and move it outside of the wrapper
```
Comment (cogniSyb) [inline blocks/images-grid/images-grid.js:26]: Remove comment and check the rest of the comments in the file

File: blocks/images-grid/images-grid.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
```
Comment (Lakshmishri) [inline blocks/images-grid/images-grid.js:1]: Can you use ```createElement ``` from script.js for creating elements. You can also pass classes to it.

**Review by Lakshmishri — CHANGES_REQUESTED** [review]
Screenshot 2023-08-01 at 09 05 47
We have carousel for mobile, are we skipping it ?

**Comment by kesiah** [comment]
> Screenshot 2023-08-01 at 09 05 47 We have carousel for mobile, are we skipping it ?

No, the slider is there but as it's not having arrows, we are using scroll-snap to scroll in mobile. it was agreed with Syb.

**Review by Lakshmishri — CHANGES_REQUESTED** [review]
Sorry I forgot, can we rename the files to start with v2, so we know this is a new block as we agreed.

v2-images-grid ?

**Review by cogniSyb — APPROVED** [review]

**Pull request merged**